src/test/scala/org/camunda/feel/impl/builtin/BuiltinFunctionsTest.scala

```diff
@@ -243,4 +243,27 @@ class BuiltinFunctionsTest
       "Assertion failure on evaluate the expression 'list contains(assert(my_list, my_list != null, \"The condition is not true\"), 2)': The condition is not true"
     )
   }
+
+  "A is blank() function" should "return Boolean" in {
+    evaluateExpression(
+      expression = """ is blank("") """
+    ) should returnResult(true)
```

Comment:
:x: Add one more test case that verifies the invocation of the function with named parameters:

```
evaluateExpression(
      expression = """ is blank(string: "") """
    ) should returnResult(true)
```

This kind of test ensures that we don't change the parameter name accidentally.